src/envoy/http/mixer/report_data.h

```diff
@@ -28,8 +28,8 @@ namespace Envoy {
 namespace Http {
 namespace Mixer {
 namespace {
-const std::string kRbacPermissivePolicyIDField = "shadow_effective_policyID";
-const std::string kRbacPermissiveRespCodeField = "shadow_response_code";
+const std::string kRbacPermissivePolicyIDField = "shadow_effective_policy_id";
+const std::string kRbacPermissiveRespCodeField = "shadow_engine_result";
```

Comment:
Could you rename kRbacPermissiveRespCodeField to kRbacPermissiveEngineResultField or something similar, so that we keep the variable name consistent with the attribute.

Reply:
renamed, PTAL.